test/unit/processing/confirm-due-dates.test.js

```diff
@@ -586,4 +590,25 @@ describe('confirm due dates', () => {
     expect(confirmedPaymentRequests[0].dueDate).toBe('10/06/2022')
     expect(confirmedPaymentRequests[0].schedule).toBe('Q2')
   })
+
+  test('should update schedule to cover remaining payments when three day quarters and two remaining with partial settlement', () => {
```

Comment:
Add a test case which tests our rounding issue

Reply:
Yes agreed.  Originally I named this based on the assumed defect cause and neglected to change it.  Fixed now.